components/org.wso2.carbon.status.dashboard.web/src/siddhi-app/AppSpecific.jsx

```diff
@@ -237,7 +237,7 @@ export default class WorkerSpecific extends React.Component {
     renderLatencyChart() {
```

Comment:
All jsx files in this PR need to be reformatted. hard coded styling need to be moved to css files

Reply:
It's better to pass styles through a const, so the code looks nice. Not recommended to move MUI styles to CSS.